<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<html>
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
   <meta name="Generator" content="Cosmo Create 1.0.3">
   <meta name="ProgId" content="Word.Document">
   <meta name="Originator" content="Microsoft Word 9">
   <meta name="GENERATOR" content="Mozilla/4.6 [en] (X11; I; SunOS 5.6 i86pc) [Netscape]">
   <title>  </title>
<link rel=File-List href="./rna_fichiers/filelist.xml">
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>lipn</o:Author>
  <o:LastAuthor>lipn</o:LastAuthor>
  <o:Revision>284</o:Revision>
  <o:TotalTime>410</o:TotalTime>
  <o:LastPrinted>2000-05-12T13:33:00Z</o:LastPrinted>
  <o:Created>2000-05-11T12:05:00Z</o:Created>
  <o:LastSaved>2000-05-12T14:15:00Z</o:LastSaved>
  <o:Pages>3</o:Pages>
  <o:Words>675</o:Words>
  <o:Characters>3849</o:Characters>
  <o:Company>lipn</o:Company>
  <o:Lines>32</o:Lines>
  <o:Paragraphs>7</o:Paragraphs>
  <o:CharactersWithSpaces>4726</o:CharactersWithSpaces>
  <o:Version>9.2812</o:Version>
 </o:DocumentProperties>
</xml><![endif]-->
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:HyphenationZone>21</w:HyphenationZone>
  <w:DisplayHorizontalDrawingGridEvery>0</w:DisplayHorizontalDrawingGridEvery>
  <w:DisplayVerticalDrawingGridEvery>0</w:DisplayVerticalDrawingGridEvery>
  <w:UseMarginsForDrawingGridOrigin/>
  <w:Compatibility>
   <w:FootnoteLayoutLikeWW8/>
   <w:ShapeLayoutLikeWW8/>
   <w:AlignTablesRowByRow/>
   <w:ForgetLastTabAlignment/>
   <w:LayoutRawTableWidth/>
   <w:LayoutTableRowsApart/>
  </w:Compatibility>
 </w:WordDocument>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;
	mso-font-charset:2;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:0 268435456 0 0 -2147483648 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
h1
	{mso-style-next:Normal;
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:center;
	mso-pagination:widow-orphan;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:20.0pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Times New Roman";
	mso-font-kerning:0pt;
	mso-ansi-language:EN-US;
	font-style:italic;}
h2
	{mso-style-next:Normal;
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan;
	page-break-after:avoid;
	mso-outline-level:2;
	font-size:14.0pt;
	font-family:Arial;
	font-style:italic;}
h3
	{mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:35.45pt;
	margin-bottom:.0001pt;
	text-align:justify;
	mso-pagination:widow-orphan;
	page-break-after:avoid;
	mso-outline-level:3;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-ansi-language:EN-US;
	font-weight:normal;
	font-style:italic;}
h4
	{mso-style-next:Normal;
	margin:0cm;
	margin-bottom:.0001pt;
	text-align:justify;
	mso-pagination:widow-orphan;
	page-break-after:avoid;
	mso-outline-level:4;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-ansi-language:EN-US;
	font-weight:normal;
	font-style:italic;}
p.MsoBodyText, li.MsoBodyText, div.MsoBodyText
	{margin:0cm;
	margin-bottom:.0001pt;
	text-align:center;
	mso-pagination:widow-orphan;
	font-size:20.0pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	font-weight:bold;}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;
	text-underline:single;}
@page Section1
	{size:595.3pt 841.9pt;
	margin:70.85pt 70.85pt 70.85pt 70.85pt;
	mso-header-margin:36.0pt;
	mso-footer-margin:36.0pt;
	mso-paper-source:0;}
div.Section1
	{page:Section1;}
 /* List Definitions */
@list l0
	{mso-list-id:326717224;
	mso-list-type:hybrid;
	mso-list-template-ids:-904594356 67895299 67895299 67895301 67895297 67895299 67895301 67895297 67895299 67895301;}
@list l0:level1
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:124.2pt;
	mso-level-number-position:left;
	margin-left:124.2pt;
	text-indent:-18.0pt;
	font-family:"Courier New";
	mso-bidi-font-family:"Times New Roman";}
@list l0:level2
	{mso-level-tab-stop:160.2pt;
	mso-level-number-position:left;
	margin-left:160.2pt;
	text-indent:-18.0pt;}
@list l0:level3
	{mso-level-tab-stop:196.2pt;
	mso-level-number-position:left;
	margin-left:196.2pt;
	text-indent:-18.0pt;}
@list l0:level4
	{mso-level-tab-stop:232.2pt;
	mso-level-number-position:left;
	margin-left:232.2pt;
	text-indent:-18.0pt;}
@list l0:level5
	{mso-level-tab-stop:268.2pt;
	mso-level-number-position:left;
	margin-left:268.2pt;
	text-indent:-18.0pt;}
@list l0:level6
	{mso-level-tab-stop:304.2pt;
	mso-level-number-position:left;
	margin-left:304.2pt;
	text-indent:-18.0pt;}
@list l0:level7
	{mso-level-tab-stop:12.0cm;
	mso-level-number-position:left;
	margin-left:12.0cm;
	text-indent:-18.0pt;}
@list l0:level8
	{mso-level-tab-stop:376.2pt;
	mso-level-number-position:left;
	margin-left:376.2pt;
	text-indent:-18.0pt;}
@list l0:level9
	{mso-level-tab-stop:412.2pt;
	mso-level-number-position:left;
	margin-left:412.2pt;
	text-indent:-18.0pt;}
@list l1
	{mso-list-id:428235321;
	mso-list-type:hybrid;
	mso-list-template-ids:-904594356 67895307 67895299 67895301 67895297 67895299 67895301 67895297 67895299 67895301;}
@list l1:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0D8;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l2
	{mso-list-id:614677758;
	mso-list-type:hybrid;
	mso-list-template-ids:-904594356 67895309 67895299 67895301 67895297 67895299 67895301 67895297 67895299 67895301;}
@list l2:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0FC;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l3
	{mso-list-id:984356857;
	mso-list-type:hybrid;
	mso-list-template-ids:-1106238868 67895311 67895321 67895323 67895311 67895321 67895323 67895311 67895321 67895323;}
@list l4
	{mso-list-id:2118789955;
	mso-list-type:hybrid;
	mso-list-template-ids:-904594356 67895297 67895299 67895301 67895297 67895299 67895301 67895297 67895299 67895301;}
@list l4:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Symbol;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
    </style>
</head>
<body link="#0000FF" vlink="#800080" lang="FR" style="tab-interval:35.4pt">

<div class=Section1>
<div class="MsoBodyText"><span lang=EN-US style='mso-ansi-language:EN-US'>A
program to predict the consensus secondary structure of a set of unaligned
RNA sequences<o:p></o:p></span></div>

<div class="MsoBodyText"><span lang=EN-US style='font-size:14.0pt;mso-bidi-font-size:
12.0pt;mso-ansi-language:EN-US;font-weight:normal'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoBodyText"><span style='font-size:12.0pt;font-weight:normal'>D.
Bouthinon (</span><span lang=EN-US style='font-size:12.0pt;mso-ansi-language:
EN-US;font-weight:normal'><span 
lang=FR style='mso-ansi-language:FR'><b><a href="mailto:db@lipn.univ-paris13.fr">db@lipn.univ-paris13.fr</span></span><span 
style='font-size:12.0pt;font-weight:normal'></a></b>)&nbsp;<o:p></o:p></span></div>

<div class="MsoBodyText"><span style='font-size:12.0pt;font-weight:normal'>H.
Soldano (</span><span style='font-size:12.0pt'><a href="mailto:soldano@lipn.univ-paris13.fr">soldano@lipn.univ-paris13.fr</span><span 
style='font-size:12.0pt;font-weight:normal'></a>)<o:p></o:p></span></div>

<div class="MsoBodyText"><span lang=EN-US style='font-size:12.0pt;mso-ansi-language:
EN-US;font-weight:normal'>LIPN-UPRESA
CNRS 7030<o:p></o:p></span></div>

<div class="MsoBodyText"><span style='font-size:12.0pt;font-weight:normal'>Universit&eacute;
Paris 13, Institut Galil&eacute;e&nbsp;<o:p></o:p></span></div>

<div class="MsoBodyText"><span style='font-size:12.0pt;font-weight:normal'>99
Avenue Jean-Baptiste Cl&eacute;ment 93430 Villetaneuse (France)<o:p></o:p></span></div>


<p class="MsoBodyText"><span style='font-size:12.0pt;font-weight:normal'><a href="http://www-lipn.univ-paris13.fr">http://www-lipn.univ-paris13.fr</a><o:p></o:p></span>
<h1 style="text-align:left">
<span style='mso-ansi-language:FR'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></h1>

<h1 style="text-align:left">
<span lang=EN-US>Introduction</span></h1>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'>We
have designed a method based on a new representation of any RNA secondary
structure as a set of structural relationships between the helices of the
structure. We refer to this representation as a structural pattern.&nbsp;<o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'>In
a first step, we use thermodynamic parameters to select, for each sequence,
the k best secondary structures according to energy minimization and we
represent each of them using its corresponding structural pattern.<o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'>In
a second step, we search for the repeated structural patterns, i.e. the
largest structural patterns that occur in at least one sequence, i.e. included
in at least one of the structural patterns associated to each sequence.
Thanks to an efficient encoding of structural patterns, this search comes
down to identifying the largest repeated word suffixes in a dictionary.<o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'>In
a third step, we compute the plausibility of each repeated structural pattern
by checking if it occurs more frequently in the&nbsp; sequences under investigation
than in random RNA sequences. We then suppose that the consensus secondary
structure corresponds to the repeated structural pattern that displays
the highest plausibility.<o:p></o:p></span></div>

<br>&nbsp;

<p class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'>For
more details see "A new method to predict the consensus secondary structure
of a set of unaligned RNA sequences", <a href="bioInfPS/">(D. Bouthinon
and H. Soldano, <i>BioInformatics</i>, Vol 15 n&deg;10, 1999, pages 785-798)</a>.<o:p>
<div class="MsoNormal"></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'>The
method described above is implemented as C programs and C-shell scripts.
Some limitations of the current implementation are listed below :<o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<ol TYPE="1" START="1">
<li class="MsoNormal" style="mso-list:l3 level1 lfo3;tab-stops:list 36.0pt">
<span 
     lang=EN-US style='mso-ansi-language:EN-US'>Dealing with RNA
sequences longer than 200 nucleotides will be time expensive (as all combinatorial
approaches based on a branch and bound algorithm)<o:p></o:p></span></li>

<li class="MsoNormal" style="mso-list:l3 level1 lfo3;tab-stops:list 36.0pt">
<span 
     lang=EN-US style='mso-ansi-language:EN-US'>In this release
we cannot consider consensus secondary structure made of more than 6 helices
(a new version will overcome this limit )<o:p></o:p></span></li>

<li class="MsoNormal" style="mso-list:l3 level1 lfo3;tab-stops:list 36.0pt">
<span 
     lang=EN-US style='mso-ansi-language:EN-US'>It is designed to
run on UNIX environment (checked on Solaris 2.5.1)<o:p></o:p></span></li>
</ol>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<h1 style="text-align:left">
<span lang=EN-US>Download</span></h1>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><a href="rna.tar">Clic
here</a> to download <i>rna.tar</i>. Then execute the command <i>tar xvf
rna.ta</i>r . This will create a <i>rna</i> directory that contains the
software to install.<o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<h1 style="text-align:left">
<span lang=EN-US>Install&nbsp;</span></h1>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'>Run
the script <i>rnainstall</i> which is in the <i>rna</i> directory, then
logout and login again. Notice that <i>rnainstall</i> will modify the <i>.cshrc</i>
file.<o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<h1 style="text-align:left">
<span lang=EN-US>Compile</span></h1>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'>Run
the script <i>compile</i> which is in the <i>rna</i> directory. You need
to have the gcc compiler.<o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<h1 style="text-align:left">
<span lang=EN-US>Run&nbsp;</span></h1>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'>The
program should be executed from the directory containing your sequences
(as an example the <i>rna </i>directory contains a <i>data</i> subdirectory
that contains the file <i>rnaSeq)</i>. The program takes RNA sequences
(inputs) and marks the occurrences of the candidate consensus structures
(outputs) in the sequences. It is made of 2 scripts <i>alea </i>and <i>rna
</i><b>that
you must run in this order</b> (although there are located in the <i>bin
</i>directory
you can run them from anywhere).<o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<ul TYPE="disc">
<li class="MsoNormal" style="mso-list:l4 level1 lfo6;tab-stops:list 36.0pt">
<span 
     lang=EN-US style='mso-ansi-language:EN-US'><i>alea</span><span 
     lang=EN-US style='mso-ansi-language:EN-US'></i>
is used to create a random model of the RNA sequences :<span style="mso-spacerun: yes">&nbsp;</span>it
takes a set of unaligned RNA sequences, randomly shuffles them and outputs
a file containing the frequencies of all possible secondary structures
(having at most 6 helices) of these random sequences.<o:p></o:p></span></li>
</ul>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:35.45pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><b>syntax&nbsp;</span><span lang=EN-US
style='mso-ansi-language:EN-US'><span style='mso-tab-count:1'></span></b><i>alea<span style="mso-spacerun: yes"></span>
N&nbsp; Mod &lt; Seq</i><span 
style="mso-spacerun: yes"></span><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:1.0cm"><span lang=EN-US style='mso-ansi-language:
EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list:
l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>N</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>
is the number of RNA sequences taken in Seq (from the beginning)<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>Mod</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>
is the name of the output file that will contain the random model of the
RNA sequences<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>Seq</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>
is the name of the input file containing the RNA sequences (see the file
<i>rnaSeq
</i>in
the <i>data</i> directory to have the format of the sequences)<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:106.35pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>


<p class="MsoNormal" style="margin-left:35.45pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><b>example&nbsp;</span><span lang=EN-US
style='mso-ansi-language:EN-US'><span style='mso-tab-count:1'></span></b><i>alea
100&nbsp;<span style="mso-spacerun: yes"></span>rnaMod &lt; rnaSeq</i><o:p></o:p></span>
<h3>
<span lang=EN-US><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></h3>

<div class="MsoNormal" style="margin-left:106.35pt"><span lang=EN-US
style='mso-ansi-language:EN-US'>where
one creates the random model of <i>100</i> RNA sequences contained in the
file <i>rnaSeq</i>. <i>rnaMod<span 
style="mso-spacerun: yes"></span></i>is
the name of the file which will contain random model.<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:35.45pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<ul TYPE="disc">
<li class="MsoNormal" style="mso-list:l4 level1 lfo6;tab-stops:list 36.0pt">
<span 
     lang=EN-US style='mso-ansi-language:EN-US'><i>rna</span><span lang=EN-US
     style='mso-ansi-language:EN-US'></i>
locates the occurrences of candidate consensus secondary structures of
the set of unaligned RNA sequences using a random model of these sequences&nbsp;<o:p></o:p></span></li>
</ul>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:35.45pt"><b>syntax&nbsp;<span 
style='mso-tab-count:1'></span></b><i>rna
N P Mod &lt; Seq > Res</i><o:p></o:p></div>

<div class="MsoNormal" style="margin-left:35.45pt"><![if !supportEmptyParas]><![endif]><o:p></o:p></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list:
l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>N</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>
(the same <i>N</i> than in <i>alea</i>) is the number of RNA sequences<span style='mso-tab-count:1'></span><span 
style='mso-tab-count:1'></span><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>P&nbsp;</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>is
the percentage of RNA sequences where any candidate consensus structure
must be present<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>Mod</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>
is the random model computed by alea<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>Seq</span><span lang=EN-US style='mso-ansi-language:EN-US'></i>(the
same <i>Seq</i> than in <i>alea</i>) is the file containing the RNA sequences<o:p></o:p></span></div>


<p class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>Res</span><span lang=EN-US style='mso-ansi-language:EN-US'></i>is
the name of the file that will contain the results<o:p></o:p></span>

<p class="MsoNormal" style="margin-left:106.35pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span>

<p class="MsoNormal" style="margin-left:35.45pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><b>example&nbsp;<span style='mso-tab-count:1'></span></span><span 
lang=EN-US style='mso-ansi-language:EN-US'></b><i>rna
100 85 rnaMod &lt; rnaSeq > rnaRes</i></span><span lang=EN-US style='mso-ansi-language:EN-US'><span 
style="mso-spacerun: yes"></span><o:p></o:p></span>

<p class="MsoNormal" style="margin-left:70.8pt;text-indent:35.35pt"><span 
lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span>

<p class="MsoNormal" style="margin-left:105.7pt;text-indent:.1pt"><span 
lang=EN-US style='mso-ansi-language:EN-US'>Compute
the candidate secondary structures that are present in at least <i>85</i>%
of <i>100</i> RNA sequences contained in <i>rnaSeq</i>, and whose frequencies
in then random model <i>rnaMod</i> are low. The results are in <i>rnaRes
</i>(see
the file <i>rnaRes&nbsp;<span 
style="mso-spacerun: yes"></span></i>in
the <i>data</i> directory to have the format of the outputs)<o:p></o:p></span>
<h1 style="text-align:left">
<span lang=EN-US><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></h1>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><i>rna&nbsp;</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>is
itself made of 2 scripts, <i>sol</i> and <i>struct</i>, that can be run
separately (<b>in this order</b>) :<o:p></o:p></span></div>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<ul TYPE="disc">
<li class="MsoNormal" style="mso-list:l4 level1 lfo6;tab-stops:list 36.0pt">
<span 
     lang=EN-US style='mso-ansi-language:EN-US'><i>sol</span><span lang=EN-US
     style='mso-ansi-language:EN-US'></i>
is used to form the sets of (possible) helices having best free energies
for
each of a set of RNA sequences. This is a very time expensive computing
using a branch and bound algorithm, so we advice to not deal with sequences
longer than 200 nucleotides.<o:p></o:p></span></li>
</ul>

<div class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:35.45pt"><b>syntax&nbsp;<span 
style='mso-tab-count:1'></span></b><i>sol
N &lt; Seq > fSol</i><o:p></o:p></div>

<div class="MsoNormal" style="margin-left:35.45pt"><![if !supportEmptyParas]><![endif]><o:p></o:p></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list:
l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>N</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>
(the same <i>N</i> than in <i>alea</i>) is the number of RNA sequences<span style='mso-tab-count:1'></span><span 
style='mso-tab-count:1'></span><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>Seq</span><span lang=EN-US style='mso-ansi-language:EN-US'></i>(the
same <i>Seq</i> than in <i>alea</i>) is the file containing the RNA sequences<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>fSol<span style="mso-spacerun: yes"></span></span><span lang=EN-US
style='mso-ansi-language:EN-US'></i>is
the name of the file that will contain the sets of possible helices<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:106.35pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>


<p class="MsoNormal" style="margin-left:35.45pt"><b>example&nbsp;<span 
style='mso-tab-count:1'></span></b><i>sol
100 &lt; rnaSeq > rnaSol</i>

<p class="MsoNormal" style="margin-left:70.8pt;text-indent:35.35pt"><![if !supportEmptyParas]><![endif]><o:p></o:p>

<p class="MsoNormal" style="margin-left:105.7pt;text-indent:.1pt"><span 
lang=EN-US style='mso-ansi-language:EN-US'>Compute
the sets of possible helices for each among <i>100</i> RNA sequences (contained
in <i>rnaSeq</i>). The results are in <i>rnaSol</i><o:p></o:p></span>

<p class="MsoNormal" style="margin-left:105.7pt;text-indent:.1pt"><span 
lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span>
<ul TYPE="disc">
<li class="MsoNormal" style="mso-list:l4 level1 lfo6;tab-stops:list 36.0pt">
<span 
     lang=EN-US style='mso-ansi-language:EN-US'><i>struct</span><span 
     lang=EN-US style='mso-ansi-language:EN-US'></i>
is used to located the occurrences of candidate consensus secondary structures
from the sets of possible helices of each of a set of RNA sequences and
from a model random of these sequences.<o:p></o:p></span></li>
</ul>

<div class="MsoNormal" style="margin-left:18.0pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:35.45pt"><b>syntax&nbsp;<span 
style='mso-tab-count:1'></span></b><i>struct&nbsp;<span style="mso-spacerun:
yes"></span>P
Mod &lt; fSol > Res</i><o:p></o:p></div>

<div class="MsoNormal" style="margin-left:35.45pt"><![if !supportEmptyParas]><![endif]><o:p></o:p></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list:
l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>P&nbsp;</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>is
the percentage of RNA sequences that must contain any candidate consensus
structure<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>Mod</span><span 
lang=EN-US style='mso-ansi-language:EN-US'></i>
is the random model computed by <i>alea</i><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>fSol</span><span lang=EN-US style='mso-ansi-language:EN-US'></i>(the
same <i>fSol</i> than in sol) is the file containing the sets of possible
helices of the RNA sequences used in <i>sol</i> (notice that <i>fSol</i>
also contains the RNA sequences)<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:124.2pt;text-indent:-18.0pt;mso-list: l0 level1 lfo10;tab-stops:list 124.2pt"><![if !supportLists]><span lang=EN-US
style='font-family:"Courier New";mso-bidi-font-family:"Times New Roman";
mso-ansi-language:EN-US'>o&nbsp;<span style='font:7.0pt "Times New Roman"'></span></span><![endif]><span lang=EN-US style='mso-ansi-language:EN-US'><i>Res</span><span lang=EN-US style='mso-ansi-language:EN-US'></i>is
the name of the file that will contain the results<o:p></o:p></span></div>


<p class="MsoNormal" style="margin-left:106.35pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span>
<h4>
<span lang=EN-US><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></h4>

<div class="MsoNormal" style="margin-left:35.45pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><b>example&nbsp;</span><span lang=EN-US
style='mso-ansi-language:EN-US'><span style='mso-tab-count:1'></span></b><i>struct
85 rnaMod &lt; rnaSol > rnaRes</i><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:35.45pt"><span lang=EN-US
style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:105.7pt;text-indent:.1pt"><span 
lang=EN-US style='mso-ansi-language:EN-US'>Compute
the candidate secondary structures from the set of possible helices contained
in <i>rnaSol</i> that are present in at least <i>85</i>% of the corresponding
RNA sequences and whose frequencies in then random model <i>rnaMod</i>
are low. The results are in <i>rnaRes </i>(see the file <i>rnaRes</i> in
the <i>data</i> directory to have the format of the outputs)<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:105.7pt;text-indent:.1pt"><span 
lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:35.45pt;text-indent:.1pt"><span 
lang=EN-US style='mso-ansi-language:EN-US'>Running
<i>struct</i>
after <i>sol</i> is interesting because it is based on a fast algorithm,
so you can make tries with different values for the parameter P.&nbsp;<o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:70.8pt;text-indent:35.35pt"><span 
lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>

<div class="MsoNormal" style="margin-left:105.7pt;text-indent:.1pt"><span 
lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></div>


<p class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span>

<p class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span>

<p class="MsoNormal"><span lang=EN-US style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span>
<center>
<p><span lang=EN-US
style='mso-ansi-language:EN-US'><![if !supportEmptyParas]><![endif]><o:p></o:p></span></center>
</div>

</body>
</html>
